src/test/rustdoc/crate-version-escape.rs

```diff
@@ -2,5 +2,4 @@
 
 #![crate_name = "foo"]
 
-// @has 'foo/index.html' '//div[@class="block version"]/p' 'Version <script>alert("hi")</script>'
-// @has 'foo/all.html' '//div[@class="block version"]/p' 'Version <script>alert("hi")</script>'
```

Comment:
Just realized: why removing the display of the version on the `all items` page?

Reply:
It never really made any sense. The version is supposed to be only on the crate page - why add it to just one other page?

Reply:
The all items page is a bit on its but as long as it's a voluntary change, it's fine. :)